src/extension/background-script/connectors/lndhub.ts

```diff
@@ -353,15 +352,31 @@ export default class LndHub implements Connector {
 
       if (axios.isAxiosError(e)) {
         const errResponse = e.response as AxiosResponse;
+
         if (errResponse?.status === 404) {
           const method = path.replace("/", "");
           throw new Error(`${method} not supported by the connected account.`);
         }
+
+        if (errResponse?.status === 401) {
+          console.warn(
+            "TOKEN MIGHT BE EXPIERED, LET'S TRY TO GET YOU LOGGED IN"
+          );
```

Comment:
```suggestion
```

as this is a normal case I would not log anything here. (we also did not use all-caps messages so far).
if we want to log something we could do a console.info() in the authorize() function?

Reply:
Removed